### DISCUSSION TITLE
Is it possible to rename pages ?

### DISCUSSION THREAD

**Discussion by Nikolasitude** [pdfarranger/pdfarranger#926] [Q&A]
Hi everyone !
Firstly, thank you so much for developping this application

Secondly, I would like to ask you if it's possible to rename pages in the UI ?
When a pdf is imported, each page has a name like this : "[pdf_file_name] page [page number]"
***Is it possible to rename a page ?***
I don't see a rename button when I right click on a page.

Otherwise is there a possibility to group multiple pages together (and maybe give it a name) ?
This makes things easier when you want to move around a document that is spread in a series of pages.
I know that we can manually select multiple page while pressing "Ctrl" but this was an idea to arrange a pdf easier.

**Thanks !**

**Comment by kbengs** [comment]
No it's not possible to rename a page in iconview and it's not possible to group pages either. :-) 

Don't know it it helps you but maybe you could use multiple instances of pdfarranger and drag pages between them (use the other instances as visual clipboards)
